asset/scss/style.scss

```
@@ -0,0 +1,381 @@
+
+    &:hover {
+      color: $primary-color;
+    }
+  }
+}
+.active-nav {
+  border-color: $white-color;
+}
+
+.uk-logo {
+  height: 13px;
+  width: 18px;
+}
+
+// Introduce
+.introduce {
+  @include bgcolor();
+  text-align: center;
+}
+.introduce__icon {
+  font-size: 8rem;
+  color: $primary-color;
+  padding: 144px 0 112px 0;
+}
+
+.introduce-title {
+  font-size: 5rem;
+}
+.introduce-description {
+  font-size: 2rem;
+  font-weight: 200;
+  line-height: 1.5;
+  width: 1080px;
+  margin: 66px auto 84px;
+}
+
+.introduce-button {
+  font-size: 1.6rem;
+  gap: 16px;
+
+  button {
+    font-weight: 700;
+    border: 2px solid $white-color;
+    background-color: transparent;
+    padding: 16px 48px;
+    border-radius: 30px;
+    margin: 0 18px;
+    cursor: pointer;
+    transition: all 0.3s;
+
+    &:hover {
+      background-color: #83bce9;
+      color: $white-color;
+      border-color: $white-color;
+    }
+  }
+  .btn-left {
+    color: $white-color;
+  }
+  .btn-right {
+    border-color: $primary-color;
+    color: $primary-color;
+  }
+}
+
+.introduce__pagination {
+  padding: 148px 0 46px;
+}
+.dot {
+  height: 20px;
+  width: 20px;
+  border: 4px solid $bg-color;
+  background-color: transparent;
+  border-radius: 50%;
+  display: inline-block;
+  margin: 0 8px;
+  outline: 3px solid $white-color;
+}
+.dot.dot-active {
+  background-color: $primary-color;
+}
+
+// Services
+.services {
+  padding: 142px 0;
+  color: $secondary-color;
+}
+.service-wrap {
+  width: 1170px;
+  margin: 0 auto;
+}
+
+.services-title {
+  font-size: 3.6rem;
+  text-align: center;
+}
+.services-description {
+  line-height: 1.5;
+  font-size: 1.6rem;
+  margin: 80px 0;
+}
+
+.services-products {
+  display: flex;
+}
+
+.products-items {
+  flex-basis: 25%;
+  text-align: center;
+  border: 1px solid #dfe5e8;
+}
+.product-title {
+  font-size: 1.8rem;
+  padding: 36px 0;
+  background-color: $color-1;
+}
+
+.product-price {
+  background-color: $color-2;
+  color: $white-color;
+  padding: 36px 0;
+
+  h4 {
+    font-size: 5rem;
+    font-weight: 200;
+  }
+  p {
+    font-size: 1.4rem;
+    margin-top: 12px;
+  }
+}
+
+.product-description {
+  background-color: $color-1;
+  li {
+    font-size: 1.4rem;
+    padding: 20px 0;
+    border-top: 1px solid #dfe5e8;
+  }
+
+  button {
+    margin: 24px 0 50px;
+    font-size: 1.2rem;
+    font-weight: 600;
+    padding: 16px 24px;
+    border: none;
+    border-radius: 30px;
+    background-color: $btn-color;
+    color: $white-color;
+    cursor: pointer;
+    transition: all 0.3s;
+
+    &:hover {
+      background-color: $color-4;
+    }
+
+    i {
+      margin-right: 6px;
+    }
+  }
+}
+
+.product-sub-description {
+  text-align: center;
+  height: 80px;
+
+  h4 {
+    font-size: 1.8rem;
+    font-weight: normal;
+    padding: 20px 0;
+  }
+  p {
+    font-size: 1.2rem;
+    opacity: 0.8;
+  }
+}
+
+.products-items:nth-child(2) {
+  .product-price {
+    background-color: $color-3;
+  }
+}
```

đổi lại cách viết selector như dưới và thêm class này vào html thì sau này khi thêm 1 item vào đầu danh sách thì sẽ dễ maitain hơn
```suggestion
  .background-color-3 {
    background-color: $color-3;
  }
```